REQUIRE

```diff
@@ -1,2 +1 @@
 julia 0.5
-Compat 0.19.0
```

Comment:
`@compat` is still used in `test/fixed_size_arrays.jl`, though it looks like that's now dead code